src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -98,7 +98,7 @@ public function getFactory()
      */
     public function setFactoryClass($factoryClass)
     {
-        trigger_error('The '.__METHOD__.' method is deprecated since version 2.6 and will be removed in 3.0. Use Definition::setFactory() instead.', E_USER_DEPRECATED);
+        trigger_error(sprintf('Definition::setFactoryClass(%s) is deprecated since version 2.6 and will be removed in 3.0. Use Definition::setFactory() instead.', $factoryClass), E_USER_DEPRECATED);
```

Comment:
the FQCN should still be used here. Using only short names is inconsistent with other deprecations and makes the message hard to understand (short class names are not unique)